charts/single-node-values.yml

```diff
@@ -0,0 +1,63 @@
+# Default values for zenko.
```

Comment:
I wouldn't copy the existing `values.yaml` as you did (or at least partially). This file doesn't contain 'default values'. It's overlayed with the default values from the Chart's `values.yaml`.